api/models/Device.ts

```diff
@@ -562,6 +566,13 @@ order by hour;
     return newDevice;
   };
 
+  Device.prototype.updateHeartbeat = async function (nextHeartbeat) {
+    return this.update({
+      lastConnectionTime: new Date(),
```

Comment:
I think Jon said that this is automatically updated when a deice uses a JWT?

Reply:
It's automatically updated for the device event-upload and recording upload endpoints, where the device token is used, but not "automatic" in general.

Reply:
(So this update is correct)